conf.d/mysql_ps.yaml.example

```diff
@@ -0,0 +1,13 @@
+init_config:
```

Comment:
This was originally two checks: standard + new performance metrics - I felt that made no sense so I unfied into one. This I can remove, shouldn't have missed it :)